crates/rolldown_plugin_manifest/src/lib.rs

```diff
@@ -42,7 +42,8 @@ impl Plugin for ManifestPlugin {
     let entry_css_reference_ids: &FxHashSet<String> = &CSS_ENTRIES_MAP;
     let mut entry_css_asset_file_names = FxHashSet::with_capacity(entry_css_reference_ids.len());
     for reference_id in entry_css_reference_ids {
-      if let Ok(file_name) = ctx.try_get_file_name(reference_id.as_str()) {
+      // TODO: The logic look like is wired, need to check
```

Comment:
Can you elaborate the weirdness? Also a typo it should be `weird`

Reply:
You mean the empty else branch? maybe we could add a `unreachable!()` to ensure the assumption.

Reply:
Here if not found the reference_id corresponding filename, it should give a error, here ignored the error.

Reply:
The original author already explained the reason? https://github.com/rolldown/rolldown/pull/3408/files#diff-9f6ad12138087b5d8707de9d1d4003210a54ece8a4647fe2fa6f8304e6f489b6R49-R50

Reply:
Yeah. I see that, but here is weird, if some reference_id is invalid, here should avoid using it to call `get_file_name`. Ignoring the error is not good way at here.

Reply:
Ok, please fix the typo, the pr is good to merge.

Reply:
I removed the controversial comment, let's moving fast to merge it.